File: AutoTestFramework/AutoTestFramework/NavigateTo.cs
using AutoTestFramework.UIElements;
using OpenQA.Selenium;

namespace AutoTestFramework
{
    public static class NavigateTo
    {
        public static void LoginFromThroughMenu(IWebDriver driver)
        {
            Menu menu = new Menu(driver); 
            TestScenariosPage tsPage = new TestScenariosPage(driver);

            menu.TestScenarios.Click();
            tsPage.LoginForm.Click();
        }
    }
}

File: ParaBankTesting/src/test/java/config/uiconfig/NavigateTo.java
package config.uiconfig;

import config.uiconfig.uielements.RightSideNavBar;
import org.openqa.selenium.WebDriver;

public class NavigateTo {
    public static void ContactThroughRightNavBar(WebDriver driver){
        RightSideNavBar navBar = new RightSideNavBar(driver);

        navBar.ContactButton.click();
    }
}
